MediaGalleryUi/view/adminhtml/web/css/source/_module.less

```diff
@@ -286,30 +286,30 @@
         }
 
         .image-details {
+            display: block;
```

Comment:
Would be better to have `.lib-vendor-prefix-display();` here and have display flex for all devices since we removed the floats? By doing this we can remove it from the media query added down below.